pkg/api/addons.go

```diff
@@ -221,7 +221,7 @@ func (cs *ContainerService) setAddonsConfig(isUpdate bool) {
 				MemoryRequests: "50Mi",
 				CPULimits:      "50m",
 				MemoryLimits:   "250Mi",
-				Image:          specConfig.KubernetesImageBase + "ip-masq-agent-amd64:v2.0.0",
```

Comment:
isn't this to allow for a different base for Mooncake? Hardcoding it means mooncake won't be able to pull it anymore